### ISSUE TITLE
support passing passwords via file

### ISSUE THREAD

**Issue by jelmd** [VictoriaMetrics/VictoriaMetrics#930]
Currently vmdb as well as vmbackup support passing passwords via CLI, only. That's a *big security issue* and has other bad side effects like the strings to pass are usually inconsistent unless weak tokens like ascii-only are used. Passing via env vars is not a good idea for similar reasons.

Thus it would be much better, if one could pass e.g. a `--pwfile=/etc/my-vmsecrets` option to the related utilities and the utility optionally encodes it as needed. The file should contain a simple KEY=LITERAL_VALUE pair per line. E.g.:
```
snapshotAuthKey=wrong+password+not+escaped
metricsAuthKey=who#knows!
httpAuth.password=who'cares?
# comment and empty lines ignored
```

**Comment by valyala** [comment]
There is a temporary workaround - to pass passwords via environment variables. See https://victoriametrics.github.io/#environment-variables for details. Passing command-line flags via environment variables is supported by all the VictoriaMetrics components - vmagent, vmalert, vmauth, etc.

**Comment by valyala** [comment]
Support for passing passwords via files has been implemented in the commit 1f105dde9836408a7d75a3fa39a3f2ba87e8e06b . This commit has been included in the [v1.97.0 release](https://docs.victoriametrics.com/CHANGELOG.html). See [these docs](https://docs.victoriametrics.com/#security) for details.
